Assets/Scripts/StateTransitions/TransitionManager.cs

```diff
@@ -52,6 +52,18 @@ public static void processTransition(int selected)
                 }
                 break;
 
+            case (GlobalStateTracker.States.ActionMenuing, null):
+                if (selected == 0) //assumed to mean healing move is selected
+                {
+                    GlobalStateTracker.battleState = GlobalStateTracker.CharacterSelect;
+                    //rather shouldn't it be CharacterSelectTransition?
+                    transitions[2].Transition(selected); //what does this mean 
```

Comment:
This line represents the specific transition which will be ran after the TransitionManager evaluates the current state of the battle scene. In your case, you will have to change the index of the transitions array to the index of the Transition you add to it.